jorgePrecich/spotify/api.rb

```diff
@@ -0,0 +1,46 @@
+class Api
+  require 'net/http'
+  require 'uri'
+  require 'json'
+  require 'pry'
+  
+  def initialize(token)
+    @url = 'https://api.spotify.com/v1' 
+    @method = 'Get'
+    @auth = auth(token)
+  end
+
+  def get(slug = '')
+    uri = get_uri(slug)
+    req = new_req(uri)
+    req["Authorization"] = @auth 
+    res(uri, req)
+  end
+
+  private
+  def get_uri(slug = '')
```

Comment:
In Ruby we don't use prefixes as `get`, `add`, `create`, `new`, `do`, `does`, `can`, etc...

Source: [Rubocop - Ruby Style Guide](https://github.com/rubocop-hq/ruby-style-guide#naming)

![image](https://user-images.githubusercontent.com/9273142/52285616-4ede4480-292c-11e9-9bc0-1cd8c0f4c074.png)

Reply:
I forgot 😅 I wil refactor